go/client/cmd_status.go

```diff
@@ -345,8 +355,36 @@ func (c *CmdStatus) outputClients(dui libkb.DumbOutputUI, clients []keybase1.Cli
 		if len(cli.Details.Desc) > 0 {
 			dstr = ", description: " + cli.Details.Desc
 		}
-		dui.Printf("    %s [pid: %d%s%s]\n", strings.Join(cli.Details.Argv, " "), cli.Details.Pid, vstr, dstr)
+		if uis, ok := mappedUIs[cli.ConnectionID]; ok {
+			dstr += ", uis: " + strings.Join(uis, ",")
```

Comment:
small nit, put a space after the comma. can this and then notifications be on a separate line?

Reply:
Sure, I removed the space to distinguish the separators between list items and items in the client desc. Separate lines look better indeed!